Resources/Prototypes/Entities/Structures/Machines/Computers/base_structurecomputers.yml

```diff
@@ -57,3 +57,6 @@
     containers:
       board: !type:Container
         ents: []
+  - type: Tag
+    tags:
+    - Hardlight
```

Comment:
So that Holograms can interact with computers... Really not ideal, though.

Reply:
They can't do that!
*The sAI can't*